rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -500,7 +490,13 @@ private void processSigning(long executionBlockNumber, BtcECKey federatorPublicK
             logger.info("Tx fully signed {}. Hex: {}", btcTx, Hex.toHexString(btcTx.bitcoinSerialize()));
             removeUsedUTXOs(btcTx);
             provider.getRskTxsWaitingForSignatures().remove(new Sha3Hash(rskTxHash));
-            provider.getRskTxsWaitingForBroadcasting().put(new Sha3Hash(rskTxHash), new ImmutablePair(btcTx, executionBlockNumber));
+            logs.add(
+                new LogInfo(
+                    contractAddress.getBytes(StandardCharsets.UTF_8),
```

Comment:
Same question as before.

Reply:
Also, it would be very nice if we could abstract `bytesFromString` (or whatever you want to call it) somewhere so that we don't end up passing in the specific encoding over and over.

Reply:
> bytesFromString (or whatever you want to call it) somewhere so that we don't end up passing in the specific encoding over and over.

I agree. There's a TypeConverter class on rpc that we can use and move to a common package.